seeds/vote-seeds.js

```diff
@@ -0,0 +1,196 @@
+const { Vote } = require('../models');
+

```

Comment:
let's change Vote and any usage of that word to Like just so it seems a little different from just-tech-news